client/app/telecom/sms/sms/templates/remove/telecom-sms-sms-templates-remove.html

```diff
@@ -1,69 +1,36 @@
-        </button>
-    </div>
-
-    <div class="modal-body"
-         data-ng-if="!TemplateRemoveCtrl.loading.removeTemplate && !TemplateRemoveCtrl.removed">
-        <h3 class="modal-title"
-            data-translate="sms_sms_templates_remove">
-        </h3>
-        <dl class="dl-horizontal">
-            <!-- NAME -->
-            <dt data-translate="sms_sms_templates_list_title_name"></dt>
-            <dd data-ng-bind="TemplateRemoveCtrl.template.name"></dd>
-
-            <!-- DATETIME -->
-            <dt data-translate="sms_sms_templates_list_title_date"></dt>
-            <dd data-ng-bind="TemplateRemoveCtrl.template.datetime | date:'shortDate'"></dd>
-
-            <!-- ACTIVITY -->
-            <dt data-translate="sms_sms_templates_list_title_type"></dt>
-            <dd data-ng-bind="TemplateRemoveCtrl.template.activity"></dd>
-
-            <!-- DESCRIPTION -->
-            <dt data-translate="sms_sms_templates_list_title_description"></dt>
-            <dd data-ng-bind="TemplateRemoveCtrl.template.description"></dd>
-        </dl>
-    </div>
-
-    <div class="modal-body text-center"
-         data-ng-if="TemplateRemoveCtrl.loading.removeTemplate && !TemplateRemoveCtrl.removed">
-        <oui-spinner></oui-spinner>
-        <h4 data-translate="sms_sms_templates_removing"></h4>
-    </div>
-
-    <div class="modal-body"
-         data-ng-if="TemplateRemoveCtrl.removed">
-        <tuc-success-drawing-check data-draw-success-check="TemplateRemoveCtrl.removed">
+<oui-modal data-heading="{{:: 'sms_sms_templates_remove' | translate }}"
+           data-primary-action="$ctrl.remove()"
+           data-primary-label="{{:: 'submit' | translate }}"
+           data-primary-disabled="$ctrl.loading.removeTemplate || $ctrl.removed"
+           data-secondary-action="$ctrl.cancel()"
+           data-secondary-label="{{:: 'cancel' | translate }}"
+           data-secondary-disabled="$ctrl.loading.removeTemplate || $ctrl.removed"
+           data-on-dismiss="$ctrl.cancel()"
+           data-loading="$ctrl.loading.removeTemplate && !$ctrl.removed">
+    <dl class="dl-horizontal"
+        data-ng-if="!$ctrl.loading.removeTemplate && !$ctrl.removed">
+        <!-- NAME -->
+        <dt data-translate="sms_sms_templates_list_title_name"></dt>
+        <dd class="word-break"
+            data-ng-bind="::$ctrl.template.name"></dd>
+
+        <!-- DATETIME -->
+        <dt data-translate="sms_sms_templates_list_title_date"></dt>
+        <dd data-ng-bind="::$ctrl.template.datetime | date:'shortDate'"></dd>
+
+        <!-- ACTIVITY -->
+        <dt data-translate="sms_sms_templates_list_title_type"></dt>
+        <dd data-ng-bind="::$ctrl.template.activity"></dd>
+
+        <!-- DESCRIPTION -->
+        <dt data-translate="sms_sms_templates_list_title_description"></dt>
+        <dd class="word-break"
+            data-ng-bind="::$ctrl.template.description"></dd>
+    </dl>
+
+    <div data-ng-if="$ctrl.removed">
```

Comment:
Bit weird to have a `h4` and nothing after it, it's like writing the title of a book and having the remaining of the page empty